lockbox-ios/lockbox-datastore/dswrapper.js

```diff
@@ -49,7 +49,7 @@ class SwiftInteropDataStore extends DataStoreModule.DataStore {
   }
 
   async unlock(scopedKey) {
-      return super.unlock(scopedKey).then( () => {
```

Comment:
good to know that webKit jscore doesn't support arrow functions.

Reply:
;D